src/rewriters/ArithmeticEqualityRewriter.h

```diff
@@ -8,10 +8,15 @@
 #include "Rewriter.h"
 
 class EqualityRewriterConfig : DefaultRewriterConfig {
+    std::unique_ptr<Map<PTRef,bool,PTRefHash>> notOkToPartition;
     LALogic & logic;
-
 public:
-    EqualityRewriterConfig(LALogic & logic): logic(logic) {}
+    EqualityRewriterConfig(LALogic & logic): logic(logic) { notOkToPartition = std::unique_ptr<Map<PTRef,bool,PTRefHash>>(new Map<PTRef,bool,PTRefHash>()); }
```

Comment:
```suggestion
    EqualityRewriterConfig(LALogic & logic): logic(logic), notOkToPartition(new Map<PTRef,bool,PTRefHash>()) {}
```
and put `notOkToPartition` after `logic` in the declaration of members.

Reply:
Ok.